programs/utilities/stat.sc

```diff
@@ -1,83 +1,370 @@
+        '%color% /%app_name% combined <combined_stat> ', 'f ｜ ', 'g Various statistics combined together', ' \n',
+    ],
+    [
+        '%color% /%app_name% print <category> <entry> [<player>]', 'f ｜ ', 'g Prints the value of a stat [of an online player]', ' \n',
+        '%color% /%app_name% hide ', 'f ｜ ', 'g Hides the scoreboard', ' \n',
+        '%color% /%app_name% show ', 'f ｜ ', 'g Shows the scoreboard', '  \n',
+        '%color% /%app_name% bots (on/off/toggle) ', 'f ｜ ', 'g A shortcut for /%app_name% settings botsIncluded', '  \n',
+        '%color% /%app_name% default_dig <dig> ', 'f ｜ ', 'g Sets the default dig type ', 'f *', '^g For server operators only', '  \n',
+        '%color% /%app_name% settings bots_included (on/off/toggle) ', 'f ｜ ', 'g Includes or excludes bots in the scoreboard', ' \n',
+        '%color% /%app_name% settings offline_digs (on/off/toggle) ', 'f ｜ ', 'g Includes or excludes digs of offline whitelisted players in the scoreboard', ' \n',
+        '%color% /%app_name% settings dig_display (on/off/toggle) ', 'f ｜ ', 'g Shows or hides digs in the player list footer', ' \n',
+        '%color% /%app_name% settings dig_display_color <hex_color> ', 'f ｜ ', 'g Changes the color of digs display for yourself (leave empty to reset)', ' \n',
+        '%color% /%app_name% settings stat_color <hex_color> ', 'f ｜ ', 'g Changes the color of the scoreboard name for everyone (leave empty to reset) ', 'f *', '^g For server operators only', ' \n'
+    ],   
+    [
+        '%color% /%app_name% settings combined_stat info <combined_stat>', 'f ｜ ', 'g Gets info about a combined statistic ', ' \n',
+        '%color% /%app_name% settings combined_stat create <name> <display_name> <category> <entries> ', 'f ｜ ', 'g Creates a combined statistic from a list of entries', 'f *', '^g For server operators only', ' \n',
+        '%color% /%app_name% settings combined_stat delete <combined_stat> ', 'f ｜ ', 'g Deletes a combined statistic ', 'f *', '^g For server operators only', ' \n\n',
+        '%color% /%app_name% carousel start ', 'f ｜ ', 'g Starts a carousel of statistics', ' \n',
+        '%color% /%app_name% carousel stop ', 'f ｜ ', 'g Stops the carousel', ' \n',
+        '%color% /%app_name% carousel interval [<seconds>] ', 'f ｜ ', 'g Gets or sets the interval of the carousel', ' \n',
+        '%color% /%app_name% carousel list ', 'f ｜ ', 'g Lists carousel entries', ' \n',
+        '%color% /%app_name% carousel add <category> <entry> ', 'f ｜ ', 'g Adds an entry to the carousel', ' \n',
+        '%color% /%app_name% carousel remove <index> ', 'f ｜ ', 'g Removes an entry from the carousel', ' \n',
+    ]
+];
 
 __config() -> {
+    'resources' -> [
+        // Display names
+        {
+        'source' -> str('https://raw.githubusercontent.com/CommandLeo/scarpet/main/resources/stat/display_names/%d.json', global_game_major_target),

```

Comment:
This URL is fine for testing, but I think later you'll have to replace CommandLeo with gnembon to get it from the actual repo. But we can do that at the end after the bugs have been fixed.

Reply:
It was not for testing, it's for production. At the end of the day where the file is hosted doesn't matter (I can confirm scarpet is able to download resources from a different repo)

Reply:
I know that as well, cos I wrote part of the code which allows it to do so. The reason I'm saying to change it to gnembon/scarpet is cos that makes it more reliable, because users will be downloading this from gnembon/scarpet, so it makes sense for all resources to be there.

Reply:
Or maybe not, we'll see if someone else has another view on it, idk.